packages/zowe-explorer/src/dataset/actions.ts

```diff
@@ -392,8 +402,13 @@ export async function createMember(
 ) {
     const options: vscode.InputBoxOptions = {
         placeHolder: localize("createMember.inputBox.placeholder", "Name of Member"),
+        validateInput: (text) => {
+            return dsUtils.validateDataSetName(text) === true
+                ? null
+                : localize("dataset.validation", "Enter valid dataset name");
```

Comment:
Should this say "Enter valid member name" instead, and perform different validation than for data sets?

Reply:
addressed in commit [44418ad](https://github.com/zowe/vscode-extension-for-zowe/pull/1977/commits/44418add0d80197dbffdd8bd4f55121ae869e315)